frontend/views/containers/proposals/ProposalTemplate.vue

```diff
@@ -117,16 +121,24 @@ export default ({
     },
     variant: {
       validator (value) {
-        return ['addMember', 'removeMember'].indexOf(value) > -1
+        return ['addMember', 'removeMember', 'changeDistributionDate'].indexOf(value) > -1
       }
     }
   },
   computed: {
     ...mapGetters([
+      'groupSettings',
+      'ourUsername',
       'groupMembersCount',
       'groupShouldPropose',
       'groupProposalSettings'
     ]),
+    canChangeDistributionDateImmediately () {
+      return this.variant === 'changeDistributionDate' && this.ourUsername === this.groupSettings.groupCreator
+    },
```

Comment:
@Silver-IT I can't help but feel that this part is somehow violating a good practice.
`ProposalTemplate.vue`, which is a template component shared by multiple existing proposal modals, now has a computed prop that is specific to one proposal modal (`DistributionDate.vue`). Plus, `DistributionDate.vue` has the exact same computed prop in there, which makes it feel the logic that uses this computed prop here should be moved to `DistributionDate.vue`

Reply:
I got your point. Actually, this is what I felt while I was working in this part, but I didn't find the better solution.
I will try to improve this part again.